src/EditorFeatures/CSharpTest/ConvertTupleToStruct/ConvertTupleToStructTests.cs

```diff
@@ -224,27 +224,12 @@ class Test
 {
     void Method()
     {
-        var t1 = new NewStruct(a: 1, b: 2);
+        var t1 = new NewStruct(A: 1, B: 2);
     }
 }
 
-internal record struct NewStruct
+internal record struct NewStruct(int A, int B)
 {
-    public int A;
-    public int B;
-
-    public NewStruct(int a, int b)
-    {
-        A = a;
-        B = b;
-    }
-
-    public void Deconstruct(out int a, out int b)
-    {
-        a = A;
-        b = B;
-    }
-
     public static implicit operator (int A, int B)(NewStruct value)
     {
         return (value.A, value.B);
```

Comment:
so we need a test that demonstrates that references to teh tuple fields get appropriately updated when teh tuple names rename to PascalCased form.

Otherwise, it looks liek this will just cause breaks, right?

Reply:
The tuple element names always stay the same.

Reply:
Updated to always maintain the tuple element casing. Anything fancier is blocked by https://github.com/dotnet/roslyn/issues/44560